lib/vagrant-service-manager/binary_handlers/cdk_openshift_binary_handler.rb

```diff
@@ -4,7 +4,7 @@ module ServiceManager
     class CDKOpenshiftBinaryHandler < ADBOpenshiftBinaryHandler
       # Default to latest stable origin oc version for CDK as it is different than
       # OSE oc version running inside CDK development environment
-      LATEST_OC_VERSION = '1.2.1'.freeze
+      LATEST_OC_VERSION = '1.4.0'.freeze
```

Comment:
So refresh my memory. For CDK 2.x we are using the community _oc_ version when asked to install _oc_ on the host. Correct?